src/executor.cpp

```diff
@@ -1046,6 +1049,14 @@ void Executor::execute74()
     displayExecutionStatus(74, std::vector<types::FunctionNumber>(), true);
 }
 
+void Executor::execute75()
+{
+    utils::writeBusProperty<std::string>(
+        "xyz.openbmc_project.State.BMC", "/xyz/openbmc_project/state/bmc0",
+        "xyz.openbmc_project.State.BMC", "RequestedBMCTransition",
+        "xyz.openbmc_project.State.BMC.Transition.Reboot");
+}
```

Comment:
add the below line  as last line in your function: 
utils::sendCurrDisplayToPanel("REBOOTING  BMC", "              ", transport);

Reply:
Updated